lua/terrortown/menus/gamemode/guide/roles.lua

```diff
@@ -0,0 +1,10 @@
+--- @ignore
+
+CLGAMEMODESUBMENU.base = "base_gamemodesubmenu"
+
+CLGAMEMODESUBMENU.priority = 99
+CLGAMEMODESUBMENU.title = "submenu_guide_roles_title"
+
+function CLGAMEMODESUBMENU:Populate(parent)
```

Comment:
Empty function

Reply:
yes, because there are no misc settings as of yet. Same as it was before this move